includes/widgets/accordion.php

```diff
@@ -110,6 +110,9 @@ protected function _register_controls() {
 				'type' => Controls_Manager::WYSIWYG,
 				'default' => __( 'Accordion Content', 'elementor' ),
 				'show_label' => false,
+				'dynamic' => [
```

Comment:
Maybe it should be a default for all text/wysiwyg controls?

Reply:
We can add it just in WYSIWYG

Reply:
@kobizz @matipojo Updated - removed dynamic registration from this PR. handled by udi in different PR